frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx

```diff
@@ -57,6 +59,18 @@ class BundleDetail extends React.Component<
         this.fetchBundleContents();
     }
 
+    componentDidUpdate() {

```

Comment:
I think it might be cleaner to put this logic as a call to `setInterval` in `componentDidMount` -- is there a particular reason why you put it in `componentDidUpdate`?

Reply:
I will try this, just had it for demo